apps/web/src/components/AddressMevLookup.tsx

```diff
@@ -64,11 +64,10 @@ const AddressMevLookup = () => {
   };
   const connectWithTwitterDisabled = isConnectingTwitter || !!supabaseSession;
 
+  const wrapperFlexClassName = "flex flex-row items-center justify-between";
```

Comment:
this has to live on `className` itself, does not make sense to have it as var, much cleasner to keep with rest of classes in one place like rest of elements 🙏